webapp/security/views.py

```diff
@@ -571,3 +571,7 @@ def cves_sitemap():
     response.headers["Cache-Control"] = "public, max-age=43200"
 
     return response
+
+
+def cves_about():
+    return flask.render_template("security/cve/about.html")
```

Comment:
If you are just using a static template, you can [make use of `template_finder` ](https://github.com/canonical/ubuntu.com/blob/main/webapp/app.py#L638)which means that simply adding a template to the `templates/cve/about.html` should work, no need for a view function.